### DISCUSSION TITLE
Won't display phpinfo() in phpinfo.php

### DISCUSSION THREAD

**Discussion by cpsource** [phcode-dev/phoenix#1804] [General]
I'm running 3.8.8-20499 (the latest) and trying to preview phpinfo.php. I get Nothing to preview! I should be able to preview php files
as well. Is there a fix?

<img width="719" alt="Nothing to preview - Screenshot 2024-08-14 102817" src="https://github.com/user-attachments/assets/bd29c5e3-d370-45cd-b5af-afc7d6aa01d3">

**Comment by abose** [comment]
Hi @cpsource please follow https://docs.phcode.dev/docs/Features/php-live-preview/ to setup live preview for php files.

**Reply by Ahmadalkakani** [reply]
![image](https://github.com/user-attachments/assets/76787ce6-7e6c-4ca8-9dec-751ae62c2a89)
This is dont work with me pleace give me help

**Reply by charlypa** [reply]
@Ahmadalkakani 
In the current Phoenic Code, you can either do a live preview using a custom server, PHP, or native HTML. not both. @abose can add more info about this issue

**Reply by Ahmadalkakani** [reply]
![image](https://github.com/user-attachments/assets/ce22e40e-04ef-4532-bf85-89d27b657cc4)
I need help running the code php

**Reply by charlypa** [reply]
@Ahmadalkakani @cpsource 

Apologies for the delay. Please follow the steps in this document: [PHP Live Preview Setup](https://docs.phcode.dev/docs/Features/php-live-preview/). Currently, Phoenix Code does not auto-configure for PHP preview, but we are working on it.
